lib/utils.bash

```diff
@@ -66,3 +69,13 @@ install_version() {
 		fail "An error occurred while installing $TOOL_NAME $version."
 	)
 }
+
+check_version() {
```

Comment:
Checking versions for install_type==version as well since asdf doesn't really format it